_templates/README.md.erb

```diff
@@ -60,6 +60,9 @@ name: Example workflow for <%= @name %> using Snyk
 on: push
 jobs:
   security:
+    permissions:
```

Comment:
Can we follow [the official documentation example](https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/uploading-a-sarif-file-to-github#example-workflow-that-runs-the-eslint-analysis-tool) for consistency, and set this to:
```md
    permissions:
      # required for all workflows
      security-events: write
      # only required for workflows in private repositories
      actions: read
      contents: read
```

Reply:
fixed